File: app/src/main/java/com/example/myappcontacts/data/repositories/contacts/IContactsRepository.java
package com.example.myappcontacts.data.repositories.contacts;

import com.example.myappcontacts.data.dao.contacts.db.ContactsModel;

import java.util.UUID;

import io.reactivex.Completable;
import io.reactivex.Single;

public interface IContactsRepository {
    Single<ContactsModel> loadContact(UUID contactId);
    Completable updateContact(ContactsModel contactsModel);
}

File: app/src/main/java/com/example/myappcontacts/data/repositories/contacts/IContactsRepository.kt
package com.example.myappcontacts.data.repositories.contacts

import com.example.myappcontacts.data.dao.contacts.db.ContactsModel
import io.reactivex.Completable
import io.reactivex.Single
import java.util.*

interface IContactsRepository{

    fun loadContact(contactId: UUID): Single<ContactsModel>

    fun updateContact(contactsModel: ContactsModel): Completable

}